docs/tutorials/ui.md

```diff
@@ -100,6 +100,25 @@ document.addEventListener('shaka-ui-load-failed', initFailed);
 ```
 
 
+#### Enabling VR
+
+To enable the playback of VR content, there are two possibilities:
+
+1. Enable via UI config:
+```js
+const config = {
+  'displayInVrMode': true
+}
+ui.configure(config);
+```
+
+2. Using HLS or DASH with fMP4 segments and in the init segment exists 'prji'
+   and 'hfov' boxes.
+
+
+Note: VR is only supported for DASH/HLS clear streams or HLS-AES stream.
```

Comment:
It looks like we also don't support `hfov` that isn't 360?
I don't know if that's a common occurrence.

Reply:
Not right now, but I'm working on adding more support. I think it's best to document it in the README.